cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,19 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    const userName = 'Alex1205@gmail.com';
```

Comment:
1. Change the name of the variable userName to email, since the email address is used for login.

Reply:
Add a username variable that contains the expected username to verify.